#! /bin/sh /usr/share/dpatch/dpatch-run
## 21_bcopy-v3opts.manpage.dpatch
#
# Subject: adds missing and new options in the bcopy utility's manpage
# Origin: vendor, 
#  http://bugs.debian.org/cgi-bin/bugreport.cgi?msg=45;filename=manpage-bcopy-v3-and-missing-opts.patch;att=5;bug=499619
# Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=499619
# Forwarded: http://sourceforge.net/mailarchive/message.php?msg_name=200912021125.44398.kern%40sibbald.com
# Author: Lucas B. Cohen <lbc@members.fsf.org>
# Reviewed-by: Kern Sibbald <kern@sibbald.com>
# Last-Update: 2009.12.3
#
# 2009.12.7 : tweaked patch to a patchlevel 2, to accomodate dpatch and git
#
# created by Lucas B. Cohen <lbc@members.fsf.org> with dpatch v2.0.31

@DPATCH@

diff -urNad a/bacula-3.0.3/manpages/bcopy.8 b/bacula-3.0.3/manpages/bcopy.8
--- a/bacula-3.0.3/manpages/bcopy.8	2009-10-18 11:10:16.000000000 +0200
+++ b/bacula-3.0.3/manpages/bcopy.8	2009-12-03 05:21:55.000000000 +0100
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH BCOPY 8 "26 May 2006" "Kern Sibbald" "Network backup, recovery and verification"
+.TH BCOPY 8 "26 November 2009" "Kern Sibbald" "Network backup, recovery and verification"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .SH NAME
@@ -36,12 +36,18 @@
 .BI \-d\  nn
 Set debug level to \fInn\fP.
 .TP
+.BI \-dt
+Print timestamp in debug output.
+.TP
 .BI \-i\  input
 Specify input Volume names (separated by '|')
 .TP
 .BI \-o\  output
 Specify output Volume names (separated by '|')
 .TP
+.BI \-p
+Proceed inspite of I/O errors.
+.TP
 .BI \-w\  directory
 Specify working directory (default \fI/tmp\fP).
 .TP
